packages/core/src/domain/organization.ts

```diff
@@ -43,3 +43,6 @@ export interface OrganizationCreate extends BaseDomainCreate {
 }
 
 export interface Organization extends BaseDomain, OrganizationCreate {}
+
+export const isCandidateImplementor = (type: OrganizationBizType) =>
```

Comment:
Regular functions please.

Also, we created the Org Biz Type as `IT vendor` and `provider` and we’re calling this `Candidate Implementor`. Let’s keep it consistent? We’re already calling facilities OBO even though we created them as `initiator-only` and `initiator-and-responder`, so the less names we throw there the easier to understand the code/logic